src/common/motech-list/list.controller.js

```diff
@@ -4,11 +4,17 @@
     angular.module('motech-common')
         .controller('MotechListController', controller);
 
-    controller.$inject = ['$scope'];
-    function controller($scope){
+    controller.$inject = ['$scope', '$timeout', 'LoadingModal'];
+    function controller($scope, $timeout, LoadingModal){
         var ctrl = this;
         ctrl.columns = [];
         ctrl.hasCollapsibleColumn = false;
+        LoadingModal.open();
+        $scope.$on('motech-list.loading', function(){
+            $timeout(function() {
```

Comment:
This function looks like it fire immediately, what is the point of wrapping it in a timeout?